packages/generator-clay/app/templates/_package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "<%= repoName %>",
+  "version": "0.0.0",
+  "description": "Metal <%= componentName %> component",
+  "license": "BSD",
+  "repository": "metal/metal-clay-components",
+  "engines": {
+    "node": ">=0.12.0",
+    "npm": ">=3.0.0"
+  },
+  "main": "lib/<%= componentName %>.js",
+  "esnext:main": "src/<%= componentName %>.js",
+  "jsnext:main": "src/<%= componentName %>.js",
+  "files": [
+    "lib",
+    "src",
+    "test"
+  ],
+  "scripts": {
+    "build": "npm run soy && webpack",
+    "soy": "metalsoy --soyDeps '../../node_modules/clay-+(button|link|icon)/src/**/*.soy'"
```

Comment:
We can just leave `" soy '': "metalsoy"` and remove the rest and configure according to the need of the component later.